tests/server/blueprints/phenomodels/test_phenomodels_views.py

```diff
@@ -237,7 +237,9 @@ def test_phenomodel_add_delete_subpanel(app, user_obj, institute_obj):
         assert updated_model["subpanels"] == {}
 
 
-def test_phenomodel_post_add_omim_checkbox_to_subpanel(app, user_obj, institute_obj, omim_checkbox):
+def test_phenomodel_post_add_omim_checkbox_to_subpanel(
+    app, user_obj, institute_obj, test_omim_term
```

Comment:
Not your fault but mine who wrote all these tests, but why not applying the GitHub action suggestion to remove the unused user_obj param to all the tests here?

Reply:
It's been removed